internal/dcache/cluster_manager/cluster_manager.go

```diff
@@ -2201,21 +2215,21 @@ func (cmi *ClusterManager) updateMVList(rvMap map[string]dcache.RawVolume,
 			// RV, so we don't reduce it now.
 			//
 			if rvMap[rvName].State != dcache.StateOffline {
-				if mv.RVs[rvName] != dcache.StateOffline {
+				if mv.RVs[rvName] != dcache.StateOffline && mv.RVs[rvName] != dcache.StateInbandOffline {
 					consumeRVSlot(mvName, rvName)
 				}
 			}
 		}
 
-		common.Assert((onlineRVs+offlineRVs+outofsyncRVs+syncingRVs) == len(mv.RVs),
+		common.Assert((onlineRVs+offlineRVs+outofsyncRVs+syncingRVs+inbandOfflineRVs) == len(mv.RVs),
```

Comment:
nit, always place inband offline right next to offline for best readabiliy.
f.e.,
common.Assert((onlineRVs+offlineRVs+inbandOfflineRVs+outofsyncRVs+syncingRVs) == len(mv.RVs),

same in all places.

also move L2181 to 2178

Reply:
Updated